packages/scene-composer/src/components/Tree/TreeItem.tsx

```diff
@@ -23,33 +28,45 @@ export interface TreeItemProps extends TreeItemInnerProps, ComponentPropsWithRef
 
 const TreeItemInner: FC<TreeItemInnerProps> = ({
   children,
+  label,
   onSelected = /* istanbul ignore next */ () => {},
   selectable = true,
   selected = false,
   className = '',
   onActivated = () => {},
 }) => {
+  const sceneComposerId = useSceneComposerId();
+  const setSelectedSceneNodeRef = useStore(sceneComposerId).getState().setSelectedSceneNodeRef;
+
+  const selectFromChildren = () => {
```

Comment:
`useCallback`?

Reply:
This is invoked directly from a function in the same component that uses the `useCallback` there shouldn't be any need for one here.